test/components/connector.spec.js

```diff
@@ -114,6 +114,14 @@ describe('Connector', () => {
     expect(receivedDispatch).toBe(store.dispatch);
   });
 
+  it('Should allow ES6 modules', () => {
+    // The tests are run in Node, so we are unable to use actual ES6 modules. We get quite
+    // close by emulating it
+    const fakeModule = new Number(1);
```

Comment:
I think having own `class FakeModule {}; const fakeModule = new FakeModule()` would be slightly less confusing. What do you think?

Reply:
Makes sense, thanks! I'll update the original branch, which will be reflected here (hopefully!)